.github/workflows/pipeline.yaml

```diff
@@ -20,13 +21,22 @@ jobs:
       GRAPH_STORE_ENDPOINT: ${{ secrets.GRAPH_STORE_ENDPOINT }}
       GRAPH_STORE_USER: ${{ secrets.GRAPH_STORE_USER }}
       GRAPH_STORE_PASSWORD: ${{ secrets.GRAPH_STORE_PASSWORD }}
+      PUBLISH_GRAPH_STORE_ENDPOINT: ${{ secrets.GRAPH_STORE_ENDPOINT }}
+      PUBLISH_GRAPH_STORE_USER: ${{ secrets.GRAPH_STORE_USER }}
+      PUBLISH_GRAPH_STORE_PASSWORD: ${{ secrets.GRAPH_STORE_PASSWORD }}
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1.4.4
         with:
           node-version: 14
       - run: yarn install --ci
-      - run: yarn ts-node index.ts transform --job "${{ github.event.inputs.job }}" --execution-url "https://${{ github.repository }}/actions/runs/${{ github.run_id }}" --to ${{ github.event.inputs.to }} --debug
+      - name: transform pipeline
+        if: github.event.inputs.transform_job != ''
+        run: yarn ts-node index.ts transform --job "${{ github.event.inputs.transform_job }}" --execution-url "https://${{ github.repository }}/actions/runs/${{ github.run_id }}" --to ${{ github.event.inputs.to }} --debug
```

Comment:
No biggie but you might extract the `https://${{ github.repository }}/actions/runs/${{ github.run_id }}` into the `env`section. These command are already long enough

Reply:
Like this https://github.com/zazuko/cube-creator/pull/269/commits/4d92b392eebf7fb28fac40d647c6f7fc410e9064 ?

Reply:
Well, almost. Still gotta pass `--execution-url` to the commands :)